com.unity.render-pipelines.high-definition/Editor/Material/Lit/BaseLitGUI.cs

```diff
@@ -189,6 +189,39 @@ static public void SetupStencil(Material material, bool receivesSSR, bool useSpl
         static public void SetupBaseLitMaterialPass(Material material)
         {
             material.SetupBaseUnlitPass();
+
+            bool emissiveIsActive = false;
+            bool emissionPassCanBeDisabled = false;
+            if (material.HasProperty(kUseEmissiveIntensity))
+            {
+                emissionPassCanBeDisabled = true;
+                var useIntensity = material.GetInt(kUseEmissiveIntensity);
+                if (useIntensity == 0)
+                {
+                    if (material.HasProperty(kEmissiveColor))
+                    {
+                        var emissionColor = material.GetColor(kEmissiveColor);
+                        if (emissionColor.r > 0.0 || emissionColor.g > 0.0 || emissionColor.b > 0.0)
+                        {
+                            emissiveIsActive = true;
+                        }
+                    }
+                }
+                else
+                {
+                    if (material.HasProperty(kEmissiveIntensity))
+                    {
+                        var intensityValue = material.GetFloat(kEmissiveIntensity);
+                        if (intensityValue > 0.0)
+                        {
+                            emissiveIsActive = true;
+                        }
+                    }
+                }
+            }
+
+            if (emissionPassCanBeDisabled)
+                material.SetShaderPassEnabled(HDShaderPassNames.s_ForwardEmissiveStr, emissiveIsActive);
```

Comment:
Did you test what happens when you create a material with a Lit shader (emission is back and the pass is disabled) and then assign a shader graph set up in a way that the emission pass is enabled?
This is a pretty common use case because when we create a material it has the lit shader by default.